runelite-client/src/main/resources/slayer_xp.json

```diff
@@ -0,0 +1,4166 @@
+  "Gargoyle": [
+    105.0,
+    111.0
+  ],
+  "Banshee": [
+    22.0,
+    23.0
+  ],
+  "Rockslug": [
+    27.0,
+    29.0
+  ],
+  "Dust devil": [
+    105.0,
+    93.0,
+    130.0,
+    110.0
+  ],
+  "Turoth": [
+    76.0,
+    83.0,
+    77.0,
+    85.0,
+    79.0,
+    87.0,
+    81.0,
+    89.0
+  ],
+  "Pyrefiend": [
+    45.0,
+    43.0
+  ],
+  "Jelly": [
+    75.0,
+    78.0
+  ],
+  "Infernal Mage": [
+    60.0,
+    66.0
+  ],
+  "Crawling Hand": [
+    16.0,
+    8.0,
+    19.0,
+    12.0
+  ],
+  "Lizard": [
+    40.0,
+    42.0
+  ],
+  "Desert Lizard": [
+    15.0,
+    12.0,
+    25.0,
+    24.0,
+    40.0,
+    42.0
+  ],
+  "Small Lizard": [
+    15.0,
+    12.0
+  ],
+  "Harpie Bug Swarm": [
+    25.0,
+    46.0
+  ],
+  "Skeletal Wyvern": [
+    210.0,
+    140.0
+  ],
+  "Killerwatt": [
+    51.0,
+    55.0
+  ],
+  "Zygomite": [
+    65.0,
+    74.0,
+    75.0,
+    86.0
+  ],
+  "Wall beast": [
+    105.0,
+    49.0
+  ],
+  "Giant frog": [
+    -1.0,
+    13.0,
+    -1.0,
+    99.0
+  ],
+  "Big frog": [
+    -1.0,
+    10.0,
+    -1.0,
+    24.0
+  ],
+  "Cave slime": [
+    25.0,
+    23.0
+  ],
+  "Cave bug": [
+    5.0,
+    6.0,
+    93.0,
+    96.0
+  ],
+  "Bloodveld": [
+    120.0,
+    76.0,
+    120.0,
+    81.0
+  ],
+  "Cave kraken": [
+    125.0,
+    127.0
+  ],
+  "Kraken": [
+    255.0,
+    291.0
+  ],
+  "Smoke devil": [
+    185.0,
+    160.0
+  ],
+  "Thermonuclear smoke devil": [
+    240.0,
+    301.0
+  ],
+  "Monk": [
+    -1.0,
+    5.0
+  ],
+  "Sorebones": [
+    22.0,
+    57.0
+  ],
+  "Zombie pirate": [
+    52.0,
+    57.0
+  ],
+  "Barrelchest": [
+    -1.0,
+    190.0,
+    -1.0,
+    380.0
+  ],
+  "Zombie swab": [
+    50.0,
+    55.0
+  ],
+  "Evil spirit": [
+    -1.0,
+    150.0
+  ],
+  "Fever spider": [
+    40.0,
+    49.0
+  ],
+  "Tyras guard": [
+    -1.0,
+    110.0
+  ],
+  "Arrg": [
+    150.0,
+    113.0,
+    299.5,
+    210.0
+  ],
+  "Ice wolf": [
+    70.0,
+    96.0,
+    70.0,
+    132.0
+  ],
+  "Ice Troll": [
+    129.0,
+    120.0,
+    97.0,
+    121.0,
+    86.0,
+    123.0,
+    86.0,
+    124.0
+  ],
+  "Goblin": [
+    5.0,
+    2.0,
+    5.0,
+    5.0,
+    7.0,
+    11.0,
+    3.0,
+    12.0,
+    15.0,
+    12.0,
+    16.0,
+    13.0,
+    13.0,
+    13.0,
+    16.0,
+    15.0,
+    22.0,
+    16.0,
+    18.0,
+    17.0,
+    26.0,
+    25.0
+  ],
+  "Damis": [
+    -1.0,
+    103.0,
+    -1.0,
+    200.0,
+    -1.0,
+    174.0,
+    -1.0,
+    272.0
+  ],
+  "Stranger": [
+    -1.0,
+    95.0
+  ],
+  "Ice troll": [
+    129.0,
+    120.0,
+    97.0,
+    121.0,
+    86.0,
+    123.0,
+    86.0,
+    124.0
+  ],
+  "Mummy": [
+    -1.0,
+    84.0,
+    -1.0,
+    -1.0,
+    -1.0,
+    103.0
+  ],
+  "Scarabs": [
+    -1.0,
+    92.0
+  ],
+  "Bandit champion": [
+    -1.0,
+    70.0
+  ],
+  "Baby Roc": [
+    50.0,
+    75.0
+  ],
+  "Giant Roc": [
+    -1.0,
+    172.0,
+    -1.0,
+    257.0
+  ],
+  "Shadow": [
+    -1.0,
+    73.0
+  ],
+  "Me": [
```

Comment:
Monsters that don't giver slayer xp shouldn't be on the list

Reply:
You should probably reply her :p But yeah, if i understand correctly, the monsters that give 0 slayer xp is never used, so there's no reason to have them on the list. And i would edit the scraper to not include them instead of running another program

Reply:
I'll edit the scraper but I'm going to run this list through a quick different program because running the scraper takes at least an hour and a quick program to delete the extra -1 xp creatures takes way less than that.

Reply:
And yeah you're understanding correctly. I agree that it does make more sense for the slayer xp json file to only include NPCs that actually can give Slayer xp